src/main/java/world/inetum/realdolmen/inetumrealJobs/payload/request/SignupRequest.java

```diff
@@ -1,28 +1,32 @@
 package world.inetum.realdolmen.inetumrealJobs.payload.request;
 
+import world.inetum.realdolmen.inetumrealJobs.dtos.AddressDto;
+import world.inetum.realdolmen.inetumrealJobs.entities.enums.Gender;
+
 import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
-import java.util.Set;
+import java.time.LocalDate;
 
 public class SignupRequest {
-
-    private String username;
-
     @Email
```

Comment:
Also annotate NotNull (or NotBlank, but that's partially covered by the Email annotation)

Reply:
Fixed